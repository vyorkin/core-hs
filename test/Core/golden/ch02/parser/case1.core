f g x = case x of
  <0> y   -> g y
  <1> w z -> z
