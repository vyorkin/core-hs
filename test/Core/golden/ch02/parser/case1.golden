[(Name "f",[Name "g",Name "x"],ECase (EVar (Name "x")) [Alter 0 [Name "y"] (EAp (EVar (Name "g")) (EVar (Name "y"))),Alter 1 [Name "w",Name "z"] (EVar (Name "z"))])]